doc/update_banners.md

```diff
@@ -0,0 +1,58 @@
+# How to Update Banners
+
+This documentation provides instructions on how to update banners for the website using the `banners.json` file located at `src/assets/data/banners.json`.
+
+## File Structure
+
+The `banners.json` file has the following structure:
+
+```json
+{
+  "top": {
+    "enable": false,
+    "content": "This banner will be displayed on top of the website."
+  },
+  "event": {
+    "enable": true,
+    "title": "Green Code Initiative I/O 2025",
+    "content": "It's an awesome event!",
+    "action": {
+      "label": "I register",
+      "link": "https://green-code-initiative.org"
+    }
+  }
+}
+```
+
+## Updating Banners
+
+### Top Banner
+
+1. **Enable/Disable the Top Banner**:
+
+   - To enable the top banner, set `"enable": true`.
+   - To disable the top banner, set `"enable": false`.
+
+2. **Update the Content**:
+   - Modify the `"content"` field with the desired HTML content.
+
+### Event Banner
```

Comment:
Same thing with event banner.
Just a screenshot can save some time to remember how this event banner is displayed and where in the page